python-backend/app/api/mines/permits/permit/models/permit.py

```diff
@@ -5,6 +5,12 @@
 from sqlalchemy.schema import FetchedValue
 from app.extensions import db
 
+from app.api.mines.permits.permit_amendment.models.permit_amendment import PermitAmendment
+#for schema creation
+from app.api.mines.permits.permit.models.permit_status_code import PermitStatusCode
+from app.api.mines.documents.models.mine_document import MineDocument
+from app.api.parties.party_appt.models.mine_party_appt import MinePartyAppointment
+
```

Comment:
Because we instantiate this model to create the schema, the loading order is stricter, this should not be true if we do global schema creation.